_includes/05-emphasis.md

```diff
@@ -0,0 +1 @@
+Write out some of your awesome attributes, and use emphasis (like bold or italics) to identify keywords, programming languages, or skills. 
```

Comment:
```suggestion
### About me
**From 2003 to 2008** I studied and successfully graduated from _art school with a degree in **Fine Arts**_. I participated in the city and regional competitions of the city of Bolshoj-Kamen, Vladivostok of the Primorsky region of Russia.  
**Since 2008**, I have been involved in *football*, participated in urban and regional competitions among young people in the city of Bolshoj-Kamen and the Primorsky region, participated in the championship among students of St. Petersburg and among the groups of companies "Power Machines." At the moment I continue to participate in small tournaments in the amateur team.  
**From 2010 to 2015**, I studied and successfully graduated from _SMTU with a degree in Design and **Manufacturing of Ship Power Plants**_.  
**In 2015**, I took an internship at *Power Machines*, after which I was hired as a design engineer. During the internship I got acquainted with the stages of production of **hydroturbo-building products**, got experience of participation in real projects. During my time at the company, I took part in the international project of the Frontera hydroelectric power plant (Chile), took an active part in corporate and sports-health activities of the company. **In 2017**, I was selected in the business systems of the company "Power Machines."  
**Since 2017**, I have been working at *ODK-Saturn*. Taking office began the project, the deadline of which required prompt implementation, which successfully coped, as a result of which the project was delivered on time. I actively take part in the company's corporate activities.  
**Today** I am fond of *programming in the language of Python*, I consider programming as a new tool in the work. With interest I start to difficult tasks, to master new directions in the profession. Routine tasks make it possible to come up with ways to optimize them. With interest I help colleagues in solving work issues, new employees in adaptation in the workplace. I am looking for opportunities to develop and improve my professional qualities.
```